### DISCUSSION TITLE
Restfox is absolutely amazing!

### DISCUSSION THREAD

**Discussion by janzheng** [flawiddsouza/Restfox#35] [General]
Hi there, I just wanted to thank you for how easy Restfox is to use. I've never liked using Postman or other similar tools, but Restfox is an absolute delight and has put me on the track of doing "API-driven design" where I make sure my APIs are correct before designing + building the interfaces for my app.

It's an absolute delight; thank you so much for building it!!

**Comment by flawiddsouza** [comment]
I'm really glad you like it. Thanks for using Restfox 😄

**Comment by mishushakov** [comment]
API-driven design is a great approach btw.
